DEV_README.md

```diff
@@ -83,8 +82,32 @@ in to this you need to add the following to your plugin.xml
     </extension>
     
     
-Step 4 - You're done!
----------------------
+### Step 4 - You're done!
+
 
 Error markers will now appear in the right places. More work can be done to add quick fixes, but
 it's much harder than adding the markers.
+
+Build against local Bnd-Lib
+----------------------------
+
+Since BndTools relies on Bnd, it is sometimes necessary to use a local version of Bnd. This dependency is configured in `cnf/build.bnd` with a declaration of `bndlib` which is referenced in all projects.
+
+The Repository for Bnd is configured with `bnd_repourl` in `gradle.properties`, located at the workspace-root. This file already contains a commented entry to point to a local distribution.
+
+	#bnd_repourl=../bnd/dist/bundles
+
+Assuming Bnd is located next to BndTools, just uncomment this line, otherwise adjust according to your needs, and release Bnd (see [documentation](https://github.com/bndtools/bnd) on how to release).
+
+Restart Eclipse or change some file in `cnf/ext` to pick-up the changes.
+
+### Alternative
```

Comment:
I don't like this alternative in here. It is only partial since it does not cover the other parts of the bnd build which are used by bndtools (embedded-repo, repository, etc.). Please remove this alternative section.